sql/catalyst/src/main/resources/gregorian-julian-rebase-micros.json

```diff
@@ -0,0 +1,2401 @@
+[ {
```

Comment:
Can we comment somewhere how to generate this JSON file for our descendants?

Reply:
see https://github.com/apache/spark/pull/28119#discussion_r404679947